sqlserver/datadog_checks/sqlserver/activity.py

```diff
@@ -32,7 +32,12 @@
     ' ',
     """\
 SELECT
-    at.transaction_begin_time,
+    CONVERT(
+        NVARCHAR, TODATETIMEOFFSET(at.transaction_begin_time, DATEPART(TZOFFSET, SYSDATETIMEOFFSET())), 126
```

Comment:
This gives us ISO8601 format as documented here https://docs.microsoft.com/en-us/sql/t-sql/functions/cast-and-convert-transact-sql?view=sql-server-ver15#date-and-time-styles

Note in the documentation they say:
```Note: For a milliseconds (mmm) value of 0, the millisecond decimal fraction value will not display. For example, the value '2012-11-07T18:26:20.000 displays as '2012-11-07T18:26:20'```

I've tested this with our backend code, and the duration is still calculated correctly